src/main/java/seedu/address/logic/parser/addcommandparser/AddParticipantCommandParser.java

```diff
@@ -20,7 +22,7 @@
 /**
  * Parses input arguments and creates a new {@link AddParticipantCommand} object.
  */
-public class AddParticipantCommandParser implements Parser<AddParticipantCommand> {
+public class AddParticipantCommandParser implements Parser<AddCommand> {
```

Comment:
Why did you change this back?

Reply:
I thought we were keeping it consistent: all parsers would take in the AddCommand abstract class you have created their argument. Or actually should I keep the specific command class?

Reply:
I see I see I don't think it matters actually. Think it's okay 👍

Reply:
Actually yea it doesn't really matter because in the end AddParticipantCommand is an AddCommand anyways. But I think you would be right in saying that we should make it more specific. I'll change it.